lib/common.go

```diff
@@ -89,6 +89,15 @@ func (a *ACBuild) lock() error {
 
 	a.lockFile, err = os.OpenFile(a.LockPath, os.O_CREATE|os.O_RDWR, 0644)
 	if err != nil {
+		switch err1 := err.(type) {
+		case *os.PathError:
+			switch err2 := err1.Err.(type) {
+			case syscall.Errno:
+				if err2 == syscall.EACCES {
+					fmt.Fprintf(os.Stderr, "you'll need to run this as a user with appropriate privileges\n")
```

Comment:
We already have errwrap in this lib, so I think it is better to wrap the original err. Libraries shouldn't directly printf to stderr.